src/applications/vaos/tests/list/video-appointments.unit.spec.jsx

```diff
@@ -265,8 +265,11 @@ describe('VAOS integration: upcoming video appointments', () => {
 
     // Should display appointment date
     expect(
-      screen.getByText((content, _element) =>
-        content.startsWith(startDate.format('dddd, MMMM D, YYYY ')),
+      screen.getByText(
+        new RegExp(
+          startDate.tz('America/Denver').format('dddd, MMMM D, YYYY'),
```

Comment:
Added a `.tz` call and converted this to a Regex, which has easier to read output in Jenkins